README.md

```diff
@@ -26,7 +26,7 @@ What does the following selector do?  `ul.dropdown > li`?
 Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
```

Comment:
http://code.tutsplus.com/tutorials/the-30-css-selectors-you-must-memorize--net-16048